src/wp-includes/functions.php

```diff
@@ -9090,3 +9090,139 @@ function wp_is_heic_image_mime_type( $mime_type ) {
+ * @param string $tooltip_text         The text content to be displayed in the tooltip.
+ * @param string $tooltip_button_label Optional. The label for the tooltip button. Default is 'Help'.
+ * @param string $position             Optional. The default position of the tooltip. Default is 'right'.
+ * 									   Accepted values: 'right', 'left', 'up' and 'down'.
+ */
+function add_tooltip( $field_id, $tooltip_text, $tooltip_button_label = 'Help', $position = 'right' ) {
+
+	if ( ! is_string( $field_id ) || '' === $field_id ) {
+		$error_arg = '$field_id';
+	} elseif ( ! is_string( $tooltip_text ) || '' === $tooltip_text ) {
+		$error_arg = '$tooltip_text';
+	} elseif ( ! is_string( $tooltip_button_label ) ) {
+		$error_arg = '$tooltip_button_label';
+	}
+
+	if ( isset( $error_arg ) ) {
+		_doing_it_wrong(
+			__FUNCTION__,
+			sprintf(
+				/* translators: %s: The invalid argument passed to add_tooltip(). */
+				__( 'The "%s" argument must be a non-empty string.' ),
+				$error_arg
+			),
+			'6.7.0'
+		);
+
+		return;
+	}
+
+	$valid_positions = array( 'top', 'right', 'bottom', 'left' );
+
+	// Validate position input.
+	if ( ! is_string( $position ) || ! in_array( $position, $valid_positions, true ) ) {
+
+		_doing_it_wrong(
+			__FUNCTION__,
+			sprintf(
+				/* translators: %s: Invalid $position argument passed to add_tooltip(). */
+				__( 'The passed "%s" argument is invalid. Default "right" position was used instead.' ),
+				'$position'
+			),
+			'6.7.0'
+		);
+
+		$position = 'right';
+	}
+
+	/**
+	 * Enqueues the Styles and Scripts for the Tooltip.
+	 *
+	 * @since 6.7.0
+	 *
+	 */
+	wp_enqueue_style( 'wp-tooltip' );
+	wp_enqueue_script( 'wp-tooltip' );
+
+	$icon_url = 'dashicons-editor-help';
+
+	/**
+	 * Filters $icon_url before the tooltip is rendered in the HTML.
+	 *
+	 * @since 6.7.0
+	 *
+	 * @param string $icon_url The URL to the icon to be used for this tooltip.
+	 *                         * Pass a base64-encoded SVG using a data URI, which will be colored to match
+	 *                         the color scheme. This should begin with 'data:image/svg+xml;base64,'.
+	 *                         * Pass the name of a Dashicons helper class to use a font icon,
+	 *                         e.g. 'dashicons-editor-help'.
+	 *                         * Pass 'none' to leave span.wp-tooltip-button-span empty so an icon can be added via CSS.
+	 * @param string $field_id The unique identifier for the tooltip container.
+	 */
+	$icon_url = apply_filters( 'wp_tooltip_icon', $icon_url, $field_id );
+	$icon_url = set_url_scheme( $icon_url );
+
+	if ( str_starts_with( $icon_url, 'dashicons-' ) ) {
+		$icon = 'class="dashicons ' . sanitize_html_class( $icon_url ) . '"';
+		$icon_html = '<span ' . $icon . '></span>';
+	} elseif ( str_starts_with( $icon_url, 'data:image/svg+xml;base64,' ) ) {
+		$icon = ' style="background-image:url(\'' . esc_attr( $icon_url ) . '\')"';
+		$icon_html = '<svg height = "20" width = "20"' . $icon . '</svg>';
```

Comment:
Add `aria-hidden` to the SVG, as well as `focusable="false"` to prevent older browser from focusing the SVG independently from the button.